src/directory/directory.mjs

```diff
@@ -1646,6 +1646,14 @@ export const directory = {
                 }
               ]
             },
+            {
```

Comment:
@katiegoines This is good to remove - we want to keep the pages but not the directory listing.